CONTRIBUTING.md

```diff
@@ -0,0 +1,150 @@
+# Contributing to Blundering Savant
+
+Thank you for your interest in contributing to the Blundering Savant project! This document provides instructions for setting up a development environment and contributing to the codebase.
+
+## Development Setup
+
+### Prerequisites
+
+1. **Go**: Install Go 1.24 or later from [https://golang.org/dl/](https://golang.org/dl/)
+2. **Git**: Ensure you have Git installed
+3. **Docker**: Install Docker for containerized development and testing
+4. **GitHub Account**: You'll need a GitHub account for testing and contributing
+
+### Local Development Environment
+
+1. **Clone the repository**:
+```bash
+git clone https://github.com/cchalm/blundering-savant.git
+cd blundering-savant
+```
+
+2. **Install development tools**:
+```bash
+# Install Just command runner
+sudo apt install just  # Ubuntu/Debian
+# or
+brew install just      # macOS
+# or see: https://github.com/casey/just#installation
+
+# Install golangci-lint for linting
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/HEAD/install.sh | sh -s -- -b $(go env GOPATH)/bin
+```
+
+3. **Set up configuration**:
```

Comment:
Remove this section and mention copying the environment template in `Environment Configuration` instead